### PR TITLE
Add indexing system test

### PR DESCRIPTION
**Pull request by olemagnusnorum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by jobergum** [comment]
Please add Copyright header to all files @olemagnusnorum

**Comment by aressem** [comment]
Great ! Successfully executed the test on my laptop in ~8 minutes.

**Review by aressem — DISMISSED** [review]

File: tests/performance/raw_indexing/raw_indexing.rb [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    end
+
+    def download_doc_file
+        container = (vespa.qrserver['0'] or vespa.container.values.first)
+        container.execute("cd #{dirs.tmpdir} && python3 #{selfdir}download_webtext.py && cd /")
+    end
+
+    def make_feed_docs
+        container = (vespa.qrserver['0'] or vespa.container.values.first)
+        container.execute("cd #{dirs.tmpdir} && python3 #{selfdir}make_json_docs.py 200000 data/webtext.train.jsonl feed_docs.json && cd /")
```
Comment (aressem) [inline tests/performance/raw_indexing/raw_indexing.rb:34]: The `cd /` at the end is not needed as these commands are executed in their own sub shells.

Comment (olemagnusnorum) [reply tests/performance/raw_indexing/raw_indexing.rb:34]: I can remove them before merging

**Review by aressem — APPROVED** [review]

**Pull request merged**